cmd/up.go

```diff
@@ -103,7 +106,22 @@ List of components that kubeapps up installs:
 			c.SkipGc = false
 		}
 
-		return c.Run(objs, wd)
+		err = c.Run(objs, wd)
+		if err != nil {
+			return err
+		}
+
+		// Generate mongodb password
+		// TODO: Do not assume keys and IDs for the secret
+		cfg, err := buildOutOfClusterConfig()
+		if err != nil {
+			return fmt.Errorf("Can not get kubernetes config: %v", err)
+		}
+		cli, err := kubernetes.NewForConfig(cfg)
+		if err != nil {
+			return fmt.Errorf("Can not get kubernetes client: %v", err)
+		}
+		return generateMongoDBSecret(cli, objs)
```

Comment:
Although we know that declarative config means the order doesn't matter, that would be better to check and create the mongoDB secret before updating/creating the rest of objects. Thus users might not be confusing of some crashloopbackoff messages